docs/docusaurus/docs/snippets/e2e_pandas_s3/06-create-checkpoint.py

```diff
@@ -0,0 +1,31 @@
+from great_expectations import get_context
+
+# TODO: will become from great_expectations import Checkpoint
```

Comment:
should be handled in followup to https://greatexpectations.atlassian.net/browse/V1-67